README.md

```diff
@@ -16,8 +16,10 @@ kalliope install --git-url https://github.com/kalliope-project/kalliope_neuron_o
 | api_key   | YES      | None    |                             | User API key of the OWM API                                                                       |
 | location  | YES      | None    |                             | The location                                                                                      |
 | lang      | No       | en      | multiple                    | First 2 letters cf : section Multilingual support in : [lang](https://openweathermap.org/current) |
-| temp_unit | No       | Kelvin  | Celsius, Kelvin, Fahrenheit |                                                                                                   |
-| country   | No       | US      | multiple                    |  Frist 2 letters of the country cf API doc                                                        |
+| temp_unit | No       | Kelvin  | Celsius, Kelvin, Fahrenheit |                                                            |                                                      
+| country   | No       | US      | multiple                    | Frist 2 letters of the country cf API doc                  |
+| day       | No       | None    | today or a weekday          | To get the forecast for a specific day of the week         |
+| days      | No       | All weekdays in english |              | To ask the day in your language you need to translate them, look at the example |
```

Comment:
should be renamed "days_translation" as it seems to be the idea. Explain that is mandatory if you use "day" value in another language than english. Also, you should deals with case sensitivity. i recommand to use lower() method on every entry the user can provide so you are sure.

Reply:
I renamed it, should be more clear now. And it iterate through the weekdays in english which are provided in the dict and I also use .lower() method.